main.py

```diff
@@ -115,32 +118,43 @@ def _restore_status(self, switches_status, interfaces_status):
                 error = ('Error while restoring interface status. The '
                          f'interface {interface_id} does not exist.')
                 raise KeyError(error)
-
+        # links
+        for link, state, in self.links_state.items():
+            try:
+                if state:
+                    self.links[link].enable()
+                else:
+                    self.links[link].disable()
+            except KeyError:
+                error = ('Error restoring link status.'
+                         f'The link {link} does not exist.')
+                raise KeyError(error)
         log.info('Network status restored.')
 
     def _load_network_status(self):
         """Load network status saved in storehouse."""
-        switches_status = {}
-        interfaces_status = {}
         status = self.storehouse.get_data()
         if status:
             switches = status.get('network_status')['switches']
+            links = status.get('network_status')['links']
```

Comment:
I didn't understand why you are using `get` for the first key. Network status not existing is an error, and should be raised. With this code, if it does not exist, a `TypeError` will be raised, instead of a `KeyError`, and `KeyError` is more meaninful.